generic_worker.go

```diff
@@ -61,11 +62,13 @@ func (g *Generic) handleErr(logger *zap.Logger, retriable bool, err error, key g
 }
 
 func (g *Generic) processKey(logger *zap.Logger, holder Holder, key gvkQueueKey) (bool /*retriable*/, error) {
+	groupKind := key.gvk.GroupKind()
```

Comment:
I wonder if we actually need group kind in the labels. Do we? This library allows a controller to process only a single GVK of objects. So GK label is always the same for metrics coming out of AppName controller. If we remove it, we probably reduce load on the Prometheus server a bit.

Reply:
This was part of the other PR - but we need the labels here because it's posting to the ctrl_process_seconds{} metric, and a single "app" may have many controllers.

Reply:
Yes, true. Thanks!